src/ToDo/Presentation/Dialogs/RenameListViewModel.cs

```diff
@@ -1,6 +1,3 @@
 namespace ToDo.Presentation.Dialogs;
 
-public partial class RenameListViewModel
-{
-    
-}
+public partial record RenameListViewModel(TaskList Entity);
```

Comment:
@dr1rrb there is something weird with this page. i cant seem to bind to "Entity.DisplayName" with ` IState<TaskList> Entity` like you would expect from other page. `<TextBlock Text={Binding Entity} />` would work, but not `.DisplayName`.

Reply:
Either way this seem to create a cleaner VM